chapter04/c04_family_call.rpy

```diff
@@ -3,7 +3,7 @@ label c04_family_call:
     c "Um... [Pause] H-How... How is your family doing?"
     k "!"
     Kenta's body tensed up at the word "family", as if ice had been thrown down his back.
-    Kenta's hazel eyes narrowed at me, looking somewhat fearful and threatening.
```

Comment:
Ah, this is an error on my part. In the very beginning of the project, back when the team only had like... 6 people in it, I had a completely different story to the one I have right now written in a novel form. Originally, Kenta was going to have _hazel eyes_. But the group found out that the original story was very similar to a Japanese song in which I explained that the way the story (back in it's planning period) had been changed to be in a similar style to the song because I was so moved by it. We talked and decided to revert the original story I had- the one you are fixing. A lot of things had been changed as well- such as Kenta's eyes. In the story, green play's a small but **big** part to the characters and their relation to Celia. Toma wears a green tank top while Meadow has a strange liking to green vegetables. Green- depending on religion- can mean "hope". So, with Kenta's eyes being green, you can say that he can _see_ hope in Celia's future and that she can overcome whatever reason she has that forces her to stay in the hospital. Something that will also be mentioned (mainly in the extra content) that Kenta's has been fused with an illegal himala stone- which is sort of like a magic stone- and causes some part of the body to change when coming in contact with other members; sort of like an ID. In Kenta's case, his eyes turn _**red**_ when in contact with other members or himala stones. Red can obviously be seen as a violent color, which represents Kenta's bloody past. If you like the story and are reading it as you edit/restruct, that's all I'm going to say for now. However, if you don't like the story and are just working because you feel obliged then let me know and I can give you some detailed information about Kenta's character and such. (Whew, sorry for the LONG comment/reply)